.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,108 @@
+name: Bug Report
+description: Submit a bug report
+labels: ['bug','status: Needs triage']
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      #### Before submitting a bug, please make sure there isn't already an [existing issue open](https://github.com/MobilityData/gtfs-validator/issues?q=is%3Aopen+is%3Aissue+label%3Abug).
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: >
+      A clear and concise description of what the bug is.
+  validations:
+    required: true
+    
+- type: textarea
+  attributes:
+    label: Steps/Code to Reproduce
+    description: |
+      Please add the steps that can reproduce the problem. We acknowledge that this step requires some effort on your side, but it really helps the project maintainers quickly reproduce the bug and understand the problem. More precise bug report tends to be solved more quickly because it avoids back-and-forth discussion between the maintainers and the reporter. 
```

```suggestion
      Please add the steps that can reproduce the problem. More precise bug report tends to be solved more quickly because it avoids back-and-forth discussion between the maintainers and the reporter. 
```
I think "We acknowledge that this step requires some effort on your side, but it really helps the project maintainers quickly reproduce the bug and understand the problem" is redundant and unnecessary - the last sentence explains why this is critical.